core/src/test/resources/application.conf

```diff
@@ -38,5 +38,10 @@ google {
       # ADC creds are NOT set up on Travis, etc.
       scheme = "mock"
     }
+    {
+      # For those integration tests that aren't (currently as of Nov 2020) run in CI that actually _do_ use ADC
+      name = "integration-test"
+      scheme = "application_default"
+    }
```

Comment:
@grsterin re:

> why this changed?

We don't run integration tests (yet?) in CI. Maybe post hackathon. The test you commented on, `IoActorProxyGcsBatchSpec` was broken by a previous commit. Since that prior commit the many, many Cromwell tests asking for "application-default" actually get the `scheme = "mock"` as listed above.

Instead of cleaning up those many, many Cromwell non-integration tests that ask for "application-default" when they really want to test with a mock, I changed this one integration test `IoActorProxyGcsBatchSpec` that when run on my laptop really needs to reach out to the internet and obtain a real auth.

IMHO some future PR should still leave this new auth mode as "integration-test", and should refactor the non-integrations tests to all use "mock" instead of "application-default".